### PR TITLE
Upgrade postcss to prep for Node 16

### PR DESCRIPTION
**Pull request by lavelle**
Also loosen the node engines range to allow testing with Node 16

### CODE REVIEW

File: packages/thumbprint-global-css/package.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
         "autoprefixer": "^9.0.1",
         "cssnano": "^4.1.0",
         "node-sass-tilde-importer": "2.0.0-alpha.1",
-        "postcss-cli": "^6.1.1",
+        "postcss": "^8.4.12",
```
Comment (danoc) [inline packages/thumbprint-global-css/package.json:29]: Why is this needed to make it work? Does the CLI have a peerDep?

**Review by danoc — APPROVED** [review]

File: packages/thumbprint-global-css/package.json (hunk 1 above)
Comment (lavelle) [reply packages/thumbprint-global-css/package.json:29]: it's not even specified as a peerDeep -- the CLI just errors if you run without it.
